test/Generator/TestAsset/ClassWithTypedProperty.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaminasTest\Code\Generator\TestAsset;
+
+final class ClassWithTypedProperty
+{
+    public string $typedProperty;
```

Comment:
While annoying, it is true that this needs a constructor declared: could you please add one?

Reply:
thanks! sure, I just pushed a fix.